docs/src/components/HomepageFeatures/index.tsx

```diff
@@ -26,6 +26,7 @@ const FeatureList: FeatureItem[] = [
     description: (
       <>
         Users can securely share decryption keys with other authenticated peers
+        or devices
```

Comment:
**suggestion**: This would imply that sharing keys with your own devices is an explicit step, which isn't (all keys are synced across devices automatically), and it's not possible to share a key to a particular device for a particular user (they also sync automatically across the recipient devices).

Reply:
ok, removed the device info.